packages/angular/README.md

```diff
@@ -46,10 +46,154 @@ import {CloudinaryImage} from '@cloudinary/url-gen/assets/CloudinaryImage';
 ```
 
 In your view add the component with your transformation
-```javascript
+```html
 <advanced-image [cldImg]="this.img"></advanced-image>
 ```
 
 ### Supported Angular Version
 - Angular 11
 - Angular 10
+
+
+Cloudinary Angular SDK
+=========================
+## About
+The Cloudinary Angular SDK allows you to quickly and easily integrate your application with Cloudinary.
+Effortlessly optimize and transform your cloud's assets.
+
+#### Note
+This Readme provides basic installation and usage information.
+For the complete documentation, see the [React SDK Guide](https://cloudinary.com/documentation/angular_integration) 
+and the [Angular SDK Reference](https://cloudinary.com/documentation/sdks/js/frontend-frameworks/AngularSDK).
+
+
+## Table of Contents
+- [Key Features](#key-features)
+- [Version Support](#Version-Support)
+- [Installation](#installation)
+- [Usage](#usage)
+    - [Setup](#Setup)
+    - [Transform and Optimize Assets](#Transform-and-Optimize-Assets)
+    - [Generate Image and HTML Tags](#Generate-Image-and-Video-HTML-Tags)
+    - [Plugins](#Advanced-Plugin-Features)
+
+## Key Features
+- [Transform](https://cloudinary.com/documentation/angular_video_manipulation#video_transformation_examples) and
+ [optimize](https://cloudinary.com/documentation/angular_image_manipulation#image_optimizations) assets.
+- Generate [image](https://cloudinary.com/documentation/angular_image_manipulation#deliver_and_transform_images) and
+ [video](https://cloudinary.com/documentation/angular_video_manipulation#video_element) tags.
+
+## Version Support
+| SDK Version   | ng 10.0  | ng 11.0 |
```

Comment:
What about Angular v12 & up?

Reply:
technically we know it works but not official just yet:)